Database/attraction.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
+
+const attractionSchema = new mongoose.Schema({
+  name: String,
+  ratings: {
+    avg: { type: Number },
+    total: { type: Number },
+  },
+  imageUrl: String,
+  price: Number,
+  description: String,
+});
+
+const attraction = mongoose.model('attraction', attractionSchema);
+
+module.exports = {
+
+  Attractions: attraction,
+
+  find: (req, res) => {
+    attraction.find({}, (err, result) => {
+      if (err) { res.send(err); } else res.send(result);
+    });
+  },
+
+  create: (req, res) => {
+    const obj = {
+      name: req.body.name,
+      ratings: [req.body.ratings],
+      imageUrl: req.body.imageUrl,
+    };
+    attraction.create(obj, (err, result) => {
+      if (err) { res.send(err); } else res.send(result);
+    });
+  },
+
+  update: (req, res) => {
+    const id = { _id: req.params.id };
+    const obj = {
+      name: req.body.name,
+      ratings: [req.body.ratings],
```

Comment:
Have you checked that this is working? Seem you be wrapping the rating in an array but isn't the schema expecting an object? I'm probably wrong since I don't know the structure of the req.body.rating value but its just a thought.